certified-connectors/USB4SAP/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+    "properties": {
+        "connectionParameters": {
+            "api_key": {
```

Comment:
@ecsadm to have a basic authentication type you can just have this connectionParameters as empty property. 
Please change to "connectionParameters": {}